plans/integration/check-variant-message/main.fmf

```diff
@@ -0,0 +1,34 @@
+discover+:
+    filter:
+        - 'tag: check-variant-message'
+provision:
+    how: libvirt
+    develop: true
+
+/centos7:
```

Comment:
Wouldn't be enough to test this on just one system (or one centos and one ol)?

Reply:
+1

Reply:
I changed it, now it's only for cent7 and ol8.